.travis.yml

```diff
@@ -44,7 +47,7 @@ env:
   - LXC_NAME=docker LXC_DISTRO=ubuntu LXC_RELEASE=artful
 
 before_install:
-  - test "${LXC_NAME}" != "docker" || docker build -t travis/algo .
+  - test "${LXC_NAME}" != "docker" && sudo modprobe wireguard || docker build -t travis/algo .
```

Comment:
Why do you modprobe wireguard directly? wg-quick calls `ip link add wg0 type wireguard`, and ip-link calls RTM_NEWLINK and that then calls request_module which finally calls modprobe itself.

Reply:
This is for to bridge the module inside LXC containers. We don't start wg-quick on the host system, only inside containers.

Reply:
Doesn't `request_module` usually execute processes in the init namespace though? (Did you try this without that line?) Mostly just curious...

Reply:
I'm not very familiar with LXC, to load the module on the host is the standard recommendation

Reply:
:+1: